pyinfra/api/host.py

```diff
@@ -218,17 +218,18 @@ def style_print_prefix(self, *args, **kwargs) -> str:
             self.print_prefix_padding,
         )
 
-    def log(self, message, log_func=logger.info):
+    def log(self, message: object, log_func: Callable[[str], Any] = logger.info) -> None:
```

Comment:
Shouldn't message be a str?

Reply:
I actually used `object` since it's what's used in https://github.com/python/typeshed/blob/main/stdlib/logging/__init__.pyi#L115